### PR TITLE
"New Skin" Labels

### PR DESCRIPTION
**Pull request by gilhsu**
image
image
image

### CODE REVIEW

File: src/components/SubTypeAccordionsContainer/index.js [hunk 1]
```diff
@@ -1,52 +1,63 @@
...
+            {fromDetailsPanel
                 && (
-                <View style={styles.typeTextContainer}>
-                  <BaseText variant="title" style={styles.typeText}>
-                    {label}
-                  </BaseText>
+                <View style={typeTextContainerStyles}>
+                  <View style={styles.typeTextPill}>
+                    <Text style={styles.typeText}>
+                      {PLURAL_RESOURCE_TYPES[type]}
```
Comment (foxtree) [inline src/components/SubTypeAccordionsContainer/index.js:25]: should this remain as `label`? 

`data`appears to be sorted by `label` (at least in some cases?^*) -- so if they've diverged, then the plumbing for `label` should be fixed?

eg: where a selector uses `sortedGroupedRecordsByType`?
image

File: src/components/Generic/BaseAccordion.js [hunk 2]
```diff
@@ -21,16 +21,20 @@ import MarkedIcon from '../Icons/MarkedIcon';
...
+    headerStyle.push({ paddingLeft: 30 });
+  }
 
   const chevronIcon = expanded
     ? <Ionicons name="chevron-up" size={16} color={Colors.accordionChevronIcon} />
     : <Ionicons name="chevron-down" size={16} color={Colors.accordionChevronIcon} />;
 
   return (
-    <View style={styles.header}>
+    <View style={headerStyle}>
```
Comment (foxtree) [inline src/components/Generic/BaseAccordion.js]: the following might be easier to read:
```
    <View style={[
       styles.header,
       (fromDetailsPanel && fromDateAccordion) ? styles.whatever : null,
    ]}>

```

File: src/components/Generic/BaseAccordion.js [hunk 3]
```diff
@@ -21,16 +21,20 @@ import MarkedIcon from '../Icons/MarkedIcon';
 import CollectionIcon from '../Icons/CollectionIcon';
 
 const AccordionHeader = ({
-  item, expanded, fromDetailsPanel, activeCollectionId,
+  item, expanded, fromDetailsPanel, activeCollectionId, fromDateAccordion,
 }) => {
   const { headerLabel, headerCount, resourceIds } = item;
+  const headerStyle = [styles.header];
+  if (fromDetailsPanel && fromDateAccordion) {
+    headerStyle.push({ paddingLeft: 30 });
```
Comment (foxtree) [inline src/components/Generic/BaseAccordion.js]: This works, without using `StyleSheet.create`?  

(eg: `styles.header` references the result of a call to `StyleSheet.create({...)}`)

**Review by foxtree — APPROVED** [review]

File: src/components/SubTypeAccordionsContainer/index.js (hunk 1 above)
Comment (gilhsu) [reply src/components/SubTypeAccordionsContainer/index.js:25]: We need this plural variant as we have a group of records with x-ResourceType. See how it's being used below. Additionally the design request is that these "pills" mimic the `ResourceTypePicker` buttons.

image

**Pull request merged**